### PR TITLE
Please review and test

### PR DESCRIPTION
**Pull request by zonesoldier**
Please review and test at 

zonesoldier.com

### CODE REVIEW

File: js/card.js [hunk 1]
```diff
@@ -570,7 +575,37 @@ cardOBJ = [
 ,[548,100,0,0,52,5,51,5,312,20,311,20,0]
 ,[549,100,0,0,52,5,51,5,302,30,301,30,0]
 ,[550,100,0,"Increase the probability of causing Sleep by an additional 2%.",155,10,60,5,0]
+,[551,3,"Bangungot",0,77,20,79,15,151,30,153,30,154,30,159,30,0]
+,[552,3,"Bakonawa","Receive 20% less damage from Boss monster.<br/>Receive 35% more damage from Non-Boss monster.<br/>For Non-Monk classes, increase physical damage against Boss monsters by 15%.<br/>For Monk classes, increase physical damage against Boss monsters by 10%.",77,20,79,-35,0]
```
Comment (Hatfun) [inline js/card.js]: Please remove all redundant message descriptions. Everything handled by the n_tok will be written in the description. Once all of the n_tok description will be written, the custom one will be added.
Here's what I see for Bakonawa:
```
Bosstype monster resistance + 20%
Resistance against normal monsters -35%
Receive 20% less damage from Boss monster.
Receive 35% more damage from Non-Boss monster.
For Non-Monk classes, increase physical damage against Boss monsters by 15%.
For Monk classes, increase physical damage against Boss monsters by 10%.
```

File: js/card.js [hunk 2]
```diff
@@ -570,7 +575,37 @@ cardOBJ = [
...
+,[553,3,"Buwaya","Receive 20% less damage from Boss monster.<br/>Receive 35% more damage from Non-Boss monster.<br/>Increase magic damage against Boss monsters by 7%.",77,20,79,-35,0]
```
Comment (Hatfun) [inline js/card.js]: Same here. I think the only very custom description you'll need is about the Non-Monk/Monk specific effects.

File: js/foot.js [hunk 3]
```diff
@@ -2608,6 +2677,10 @@ n_A_MaxHP += SkillSearch(156) * 200;
 	if(n_A_HEAD_DEF_PLUS > 6 && EquipNumSearch(1628)){
 		w += 1 * (n_A_HEAD_DEF_PLUS - 6);
 	}
+	//Buwaya Card - Matk + 7% against boss.
+	if(n_B[19] == 1){
+		w += 7;
+	}
```
Comment (Hatfun) [inline js/foot.js]: There's no mention of Buwaya card here, you're just checking if it's against boss, so you just apply w += 7 against boss regardless of equipment

Comment (zonesoldier) [reply js/foot.js]: Nice catch!

File: js/card.js (hunk 2 above)
Comment (zonesoldier) [reply js/card.js]: Will do!

File: js/card.js (hunk 1 above)
Comment (zonesoldier) [reply js/card.js]: Can do!

**Comment by zonesoldier** [comment]
Changes made, closing pull.

**Pull request closed without merging**